README.rst

```diff
@@ -124,16 +124,20 @@ Depending on your OS and environment, some of these packages may be more challen
 or any issue regarding the above packages, please check their respective documentation.
 
 
-Having trouble with PySCF?
+Quantum Chemistry Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tangelo can be used without having a quantum chemistry package installed but many algorithms, by default, depend on one being installed.
 
-Although PySCF is installed by default, this behaviour can be turned off by defining the environment variable `NO_PYSCF=1` and then
-installing using pip by `pip install git+https://github.com/goodchemistryco/Tangelo.git@main` or using the "From source, using setuptools"
-method. 
+The two quantum chemistry packages that are natively supported are `PySCF <https://pyscf.org/>`_ and `Psi4 <https://psicode.org/>`_.
 
-The classical solvers `CCSDSolver`, `MP2Solver`, `FCISolver` and `MINDO3Solver` as well as `DMETProblemDecomposition` require 
-PySCF at this time.
+`DMETProblemDecomposition` is only supported with `PySCF`.
+`SemiEmpircalSolver` requires PySCF  along with 'python3 -m pip install git+https://github.com/pyscf/semiempirical'.
```

Comment:
I think we can delete that too. It's easy to figure out, I believe we handle it with ModuleNotFound, and it's only relevant for people who actually call the Semi-Empirical solver and... to my knowledge that's no one so far. Let's not get our already-crowded README more cluttered with minor things.